diff --git a/settings-plugin/src/main/kotlin/com/facebook/react/ReactSettingsExtension.kt b/settings-plugin/src/main/kotlin/com/facebook/react/ReactSettingsExtension.kt
index 6d4f60e6fb48cae86610b6f72e8f915738634669..43c6f0d1ef32ed79805cbd6210ed2c927a69a146 100644
--- a/settings-plugin/src/main/kotlin/com/facebook/react/ReactSettingsExtension.kt
+++ b/settings-plugin/src/main/kotlin/com/facebook/react/ReactSettingsExtension.kt
@@ -39,7 +39,7 @@ abstract class ReactSettingsExtension @Inject constructor(val settings: Settings
    */
   @JvmOverloads
   public fun autolinkLibrariesFromCommand(
-      command: List<String> = listOf("npx", "@react-native-community/cli", "config"),
+      command: List<String> = listOf("npx", "-s", "@react-native-community/cli", "config"),
       workingDirectory: File? = settings.layout.rootDirectory.dir("../").asFile,
       lockFiles: FileCollection =
           settings.layout.rootDirectory
