diff --git a/fonts.gradle b/fonts.gradle
index 951394b0d4609ab57f15bd967f9c1d649a1faf93..2b288a21359b8a507873c24b90d3b62c9f8e8db1 100644
--- a/fonts.gradle
+++ b/fonts.gradle
@@ -32,6 +32,11 @@ afterEvaluate {
         lintVitalAnalyzeTask.dependsOn(fontCopyTask)
         }
       
+        def lintVitalReportModelTask = tasks.findByName("generate${targetName}LintVitalReportModel")
+        if (lintVitalReportModelTask) {
+            lintVitalReportModelTask.dependsOn(fontCopyTask)
+        }
+
         def generateAssetsTask = tasks.findByName("generate${targetName}Assets")
         generateAssetsTask.dependsOn(fontCopyTask)
       }
