diff --git a/lib/module/reanimated2/PlatformChecker.js b/lib/module/reanimated2/PlatformChecker.js
index 812f2e817d4af2c1695cc9ddd063fc56e6d7d159..a7956708e23285019933923c891055056de21f8e 100644
--- a/lib/module/reanimated2/PlatformChecker.js
+++ b/lib/module/reanimated2/PlatformChecker.js
@@ -6,7 +6,12 @@ import { Platform } from 'react-native';
 // and this file causes type errors on `global` access.
 
 export function isJest() {
-  return !!process.env.JEST_WORKER_ID;
+  // don't crash if `process` is undefined as this causes problems on web
+  try {
+    return !!process.env.JEST_WORKER_ID;
+  } catch (e) {
+    return false
+  }
 }
 
 // `isChromeDebugger` also returns true in Jest environment, so `isJest()` check should always be performed first
